spec/models/comment_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+require 'support/factory_girl'
+
+describe Comment do
+
+  let(:user){build(:user)}
+  let(:post_comment){build(:post_comment)}
+
+  it "can be added to a post" do
+    expect(post_comment.commentable).to be_a(Post)
+  end
+
+  it "requires a user_id" do
+    comment = build(:post_comment, user_id: nil)
```

Comment:
because `comment` was not defined, you are building a new object, without a body, or any other properties. Is it not valid because the `commentable_id` is nil, or because the body is?